src/ImageSharp/Quantizers/WuQuantizer{TPixel}.cs

```diff
@@ -328,12 +329,12 @@ private static float Volume(Box cube, long[] moment)
         /// <param name="direction">The direction.</param>
         /// <param name="moment">The moment.</param>
         /// <returns>The result.</returns>
-        private static long Bottom(Box cube, int direction, long[] moment)
+        private static long Bottom(ref Box cube, int direction, long[] moment)
         {
             switch (direction)
             {
                 // Red
-                case 0:
+                case 3:
```

Comment:
Both the original C codebase and the C# port use ABGR ordering in their calculations. I changed it to RGBA in this one ages ago but that complicates debugging so I changed it back.